files/build_templates/mgmt-framework.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Management Framework container
 Requires=swss.service
-After=swss.service
+After=swss.service syncd.service
```

Comment:
Can it be just "After syncd.service" as syncd is after swss ?

Reply:
Technically, yes, but it can't hurt to be more explicit.